docs/src/grigoriev/grigoriev.md

```diff
@@ -63,47 +69,46 @@ Let's generate a triangulated grid located at the appropriate coordinates:
 # Specify a resolution:
 RES_Y = (MAX_Y-MIN_Y)/30.0
 RES_X = RES_Y
+
 # Generate the mesh with appropriate dimensions and resolution:
-s′ = triangulated_grid(
-                       MAX_X-MIN_X, MAX_Y-MIN_Y,
-                       RES_X, RES_Y, Point3D)
+s = triangulated_grid(MAX_X-MIN_X, MAX_Y-MIN_Y, RES_X, RES_Y, Point3D)
+
 # Shift it into place:
-s′[:point] = map(x -> x + Point3D(MIN_X, MIN_Y, 0), s′[:point])
-s = EmbeddedDeltaDualComplex2D{Bool, Float64, Point3D}(s′)
-subdivide_duals!(s, Barycenter())
+s[:point] = map(x -> x + Point3D(MIN_X, MIN_Y, 0), s[:point])
+sd = EmbeddedDeltaDualComplex2D{Bool, Float64, Point3D}(s)
+subdivide_duals!(sd, Barycenter())
 
 fig = Figure()
 ax = CairoMakie.Axis(fig[1,1])
-wf = wireframe!(ax, s)
+wf = wireframe!(ax, sd)
 display(fig)
```

Comment:
This is not rendering

Reply:
Yeah you're right, the problem was from even before. I'll replace this with the image save - markdown display trick.

Reply:
This generates correctly now.